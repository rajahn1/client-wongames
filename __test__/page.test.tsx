import Home from "@/app/page";
import { render } from "@testing-library/react";

it("renders page", () => {
  const { container } = render(<Home />);
  expect(container).toContainHTML("Hello World");
});
