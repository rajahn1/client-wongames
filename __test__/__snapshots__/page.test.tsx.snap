// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders homepage unchanged 1`] = `
<div>
  <div>
    <h2>
       Hello World
    </h2>
  </div>
</div>
`;
